workbench/src/renderer/components/ResourcesLinks/index.jsx

```diff
@@ -3,36 +3,40 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { MdOpenInNew } from 'react-icons/md';
 
+import UG_ROOT from '../../userguideURL';
 import { ipcMainChannels } from '../../../main/ipcMainChannels';
 
 const { ipcRenderer } = window.Workbench.electron;
 
-const UG_ROOT = 'http://releases.naturalcapitalproject.org/invest-userguide/latest';
 const FORUM_ROOT = 'https://community.naturalcapitalproject.org';
 
 // map model names to forum tags:
 const FORUM_TAGS = {

```

Comment:
Instead of hard-coding this mapping, could we derive the forum tag from the model name? (in most cases, just replace `_` with `-`). Or at least default to that and only hard-code the exceptions. Also consider retroactively changing the tags that don't follow the pattern.

Reply:
We could. The only other thing to consider is that some models don't have forum tags because there are no questions. By leaving them out of this map completely, we let the link default to the forum homepage for those models instead of going to a tag that doesn't exist, or a tag that does exist but has no threads (maybe the latter would be okay). 

I think we should whatever we think is lowest maintenance. Maybe that is to proactively create tags for all models following a strict pattern and then remove the hardcoded tags like you suggest?

Also, crop productions - they share a tag which I think is okay, they also share a UG and many of the questions pertain to both models.

Reply:
Okay, lets leave it as it is